paasta_tools/cli/cmds/status.py

```diff
@@ -464,6 +467,33 @@ def print_kubernetes_status(
     return 0
 
 
+def print_tron_status(
+    service: str,
+    instance: str,
+    output: List[str],
+    tron_status,
+    verbose: int = 0,
+) -> int:
+    output.append(f"    Tron job: {tron_status.job_name}")
+    if verbose:
+        output.append(f"      Status: {tron_status.job_status}")
+        output.append(f"      Schedule: {tron_status.job_schedule}")
+    output.append("      DashBoard: {}".format(PaastaColors.blue(tron_status.job_url)))
```

Comment:
Dashboard

Reply:
?

Reply:
It should not have a capital "B" (should be "b").